### PR TITLE
Manage cucumber steps

### PR DESCRIPTION
**Pull request by lacell75**
Async codemod improvements for cucumber steps.

### CODE REVIEW

File: test/__fixtures__/async/source/spec.js [hunk 1]
```diff
@@ -160,3 +160,23 @@ const getColumns = (foo, bar) => {
...
+        },
+        (res) => {
+          if (res) {
+            resolve(true);
+        }
+        resolve(false);
+        }
+      );
+    });
+}
```
Comment (christian-bromann) [inline test/__fixtures__/async/source/spec.js:182]:
```suggestion
async function requestAsync() {
  return new Promise(resolve => {
    request({
      url: 'https://api.github.com',
      method: 'POST',
      headers: {
        accept: 'application/json',
      },
    }, (res) => resolve(res ? true : false));
  });
}
```

File: test/__fixtures__/async/transformed/spec.js [hunk 2]
```diff
@@ -164,3 +164,23 @@ const getColumns = async (foo, bar) => {
...
+        },
+        (res) => {
+          if (res) {
+            resolve(true);
+        }
+        resolve(false);
+        }
+      );
+    });
+}
```
Comment (christian-bromann) [inline test/__fixtures__/async/transformed/spec.js:186]:
```suggestion
async function requestAsync() {
  return new Promise(resolve => {
    request({
      url: 'https://api.github.com',
      method: 'POST',
      headers: {
        accept: 'application/json',
      },
    }, (res) => resolve(res ? true : false));
  });
}
```

File: test/runner.js [hunk 3]
```diff
@@ -58,7 +59,10 @@ async function runTest (framework, tests, parser = 'babel') {
             [srcFile],
             {
                 verbose: 2,
-                parser
+                parser,
+                printOptions : {
+                    lineTerminator:"\n"
```
Comment (christian-bromann) [inline test/runner.js:64]:
```suggestion
                    lineTerminator: '\n'
```

**Review by christian-bromann — APPROVED** [review]
Awesome work, some minor formatting!

**Comment by lacell75** [comment]
👍

**Pull request merged**